DigitalOffice/Services/FileService/Models/FileRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace FileService.Models
+{
+    public class FileRequest
```

Comment:
Maybe `FileCreateRequest` ?

Reply:
Nah. All Request models are kinda the same, for now at least.

Could you name a reason to have different model names?

Reply:
If we want to get a file, it would be logical to call the request getFileRequest if we want to createFileRequest

Reply:
Alright I'll rename the model,
fixed